SabreTools.DatTools/Rebuilder.cs

```diff
@@ -239,8 +250,33 @@ public class Rebuilder
                         bool rebuilt = RebuildGenericHelper(datFile, file, outDir, quickScan, date, inverse, outputFormat, asFiles);
 
                         // If we are supposed to delete the file, do so
-                        if (delete && rebuilt)
-                            File.Delete(input);
+                        if (delete && rebuilt) 
+                        {
+                            try 
+                            {
+                                File.SetAttributes(file, FileAttributes.Normal);
+                                File.Delete(file);
+                                // sometimes the file is in a directory (TOSEC), delete directory
```

Comment:
I think I understand why you want to add this change, but I'm hesitant to delete the parent directory. The only time that I would think that this is useful is if you guarantee that the file is in its own directory with no other files. This code does not do that. It just assumes it's by itself.